src/EditorFeatures/Core/DocumentationComments/CopilotGenerateDocumentationCommentProvider.cs

```diff
@@ -239,6 +240,8 @@ private static async Task<IReadOnlyList<ProposedEdit>> GetProposedEditsAsync(
 
             static string AddNewLinesToCopilotText(string copilotText, string? indentText, int characterLimit)
             {
+                // Double check that the resultant from Copilot does not produce any strings containing new line characters.
+                copilotText = Regex.Replace(copilotText, @"\r?\n", " ");
```

Comment:
Could we add tests to cover some basic scenarios?

Reply:
Yeah will do in a follow-up!